twint/tweet.py

```diff
@@ -215,7 +215,7 @@ def Tweet(tw, config):
     except KeyError:
         t.hashtags = []
     # don't know what this is
-    # t.cashtags = [cashtag.text for cashtag in tw.find_all("a", "twitter-cashtag")]
+    t.cashtags = [cashtag for cashtag in cashtag_pattern.findall(tw['full_text'])]
```

Comment:
simply replace this line with this: `t.cashtags = [cashtag['text'] for cashtag in tw['entities']['symbols']]`
this is because the data we fetch from twitter already contains thes _cashtags__ in the JSON Object.
all the above code (regex.py and import) won't be required

Reply:
done (for those following along)